generators/guestcontainer/index.js

```diff
@@ -62,6 +62,25 @@ module.exports = generators.Base.extend({
                     when: !this._isOptionSet('instanceCount')
                 },
 
+                {
+                    name: 'portMapContainer',
+                    message: 'Enter the port you want to expose:',
+                    when: !this._isOptionSet('portMapContainer')
+                },
+
+                {
+                    name: 'portMapHost',
+                    message: 'Enter the port you want to map to on the host machine:',
+                    when: !this._isOptionSet('portMapHost')
+                },
+
+                {
+                    name:'serviceEndPointName',
```

Comment:
So what is the experience now, will I be prompted for all three questions here?

Reply:
Yes, He will be prompted by the three questions 1. Port to expose on container 2. To which port to map on the host machine 3. endpoint name.

Reply:
And what happens if he doesnt want to configure port mapping? He is still prompted with three questions? Doesnt sound like a good experience..

Reply:
Should i ask him the question whether he wants to configure port mapping? And based on that i can ask for further questions. This should be good right ?

Reply:
Update the code as per our offline discussion. User will now only be asked one extra question 1. Port mapping of format container-port:host-port